generators/client/templates/angular/test/e2e/administration.spec.ts.ejs

```diff
@@ -0,0 +1,130 @@
+<%#
+ Copyright 2013-2020 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+
+6.8.0 CHANGED
+-%>
+import { browser, element, by, ExpectedConditions as ec } from 'protractor';
+
+import { NavBarPage, SignInPage } from '../page-objects/jhi-page-objects';
+<%_
+let elementGetter = `getText()`;
+if (enableTranslation) {
+    elementGetter = `getAttribute('jhiTranslate')`;
+}
+_%>
+
+const expect = chai.expect;
+
+describe('administration', () => {
+    let navBarPage: NavBarPage;
+    let signInPage: SignInPage;
+
+    before(async () => {
+        await browser.get('/');
+        navBarPage = new NavBarPage(true);
+        signInPage = await navBarPage.getSignInPage();
+        <%_ if (authenticationType !== 'oauth2') { _%>
+        await signInPage.autoSignInUsing('admin', 'admin');
+        <%_ } else { _%>
+        await signInPage.loginWithOAuth('admin', 'admin');
```

Comment:
In the main generator, we have these set up as variables that can be overridden by environment variables.

Reply:
Do you refers to: https://github.com/jhipster/generator-jhipster/blob/51574da5dba2e8c8932c6d703ce6bed92476502e/generators/client/templates/angular/src/test/javascript/e2e/admin/administration.spec.ts.ejs#L34 ? .  I have not used that because in the typeorm seed there is also an admin and system users that have admin permissions: https://github.com/jhipster/generator-jhipster-nodejs/blob/515387e91e58752c774a446d25f26701a8bad713/generators/server/templates/server/src/migrations/1570200490072-SeedUsersRoles.ts.ejs#L82 .

If we allows to pass in the test a process.env.E2E_USERNAME variable that replace the admin login, the tests will be in the part that performs the success admin login: https://github.com/jhipster/generator-jhipster/blob/51574da5dba2e8c8932c6d703ce6bed92476502e/generators/client/templates/angular/src/test/javascript/e2e/account/account.spec.ts.ejs#L81

Reply:
It's possible these tests need to be fixed. The reason we allow overriding the credentials is so you can run these tests against another identity provider, like Okta. When I first added support for them, all the tests passed. The last time I tried, they did not.

Reply:
Remember that this blueprint uses the jhipster 6.8.0 version. Now I have only random errors for the entity client e2e tests, but for the admin and account tests is all ok.